packages/actor-query-parse-sparql/lib/ActorQueryParseSparql.ts

```diff
@@ -30,14 +32,13 @@ export class ActorQueryParseSparql extends ActorQueryParse {
     const parser = new SparqlParser({
       prefixes: this.prefixes,
       baseIRI: action.baseIRI,
-      sparqlStar: true,
-      factory: dataFactory,
+      dataFactory: <DataFactory<RDF.BaseQuad>> dataFactory,
     });
     const parsedSyntax = parser.parse(action.query);
-    const baseIRI = parsedSyntax.type === 'query' ? parsedSyntax.base : undefined;
+    const baseIRI = ('type' in parsedSyntax && parsedSyntax.type === 'query') ? parsedSyntax.base : undefined;
     return {
       baseIRI,
-      operation: translate(parsedSyntax, {
+      operation: translate(<any> parsedSyntax, {
```

Comment:
Can we add a `// TODO: ...` here explaining that this is temporary, until sparqlalgebrajs has been ported to the traqula monorepo?